cmd2/rl_utils.py

```diff
@@ -129,10 +129,10 @@ def pyreadline_remove_history_item(pos: int) -> None:
             import ctypes
 
             readline_lib = ctypes.CDLL(readline.__file__)
-        except AttributeError:  # pragma: no cover
+        except (AttributeError, OSError):  # pragma: no cover
```

Comment:
We shoudl consider creating a new issue to add PyPy unit testing as part of our normal CI via GitHub Actions.  However, PyPy is unique in that code coverage analysis in it is awfully slow, so we would need to run unit tests differently than all other platforms where just the tests run and not code coverage.

Reply:
The biggest hurdle is getting cmd2 to support their version of readline.